css-brochure-form/styles.css

```diff
@@ -0,0 +1,49 @@
+body {
+  width: 50%;
```

Comment:
You should definitely not set `width` on the `body` element, the `body` should be allowed to take whatever space is necessary. Instead, I would set this width on the `form` element, since that is a boundary holding in all of your content already.